Cargo.toml

```diff
@@ -11,7 +11,14 @@ keywords = ["clipboard", "wayland"]
 
 [dependencies]
 sctk = { package = "smithay-client-toolkit", version = "0.12", default-features = false }
-wayland-client = { version = "0.28", features = ["dlopen"] }
+wayland-client = { version = "0.28", features = ["use_system_lib"] }
 
-[dev-dependencies]
-sctk = { package = "smithay-client-toolkit", version = "0.12"}
+[dev-dependencies.sctk]
+package = "smithay-client-toolkit"
+version = "0.12"
+default-features = false
+features = ["calloop", "frames"]
```

Comment:
I don't think `smithay-clipboard` needs the `frames` feature of SCTK, for what it's worth.

Reply:
The library compiles fine without it, but the clipboard example/demo application does not:

```
error[E0432]: unresolved import `sctk::window::ConceptFrame`
 --> examples/clipboard.rs:6:20
  |
6 | use sctk::window::{ConceptFrame, Event as WindowEvent};
  |                    ^^^^^^^^^^^^ no `ConceptFrame` in `window`
```

Also this is is in dev-dependencies and not regular runtime dependencies so I don't think it's a big deal.

Reply:
Ah right, my bad!